### ISSUE TITLE
@nrshapiro I have finished work for the day, but I just had a random look at a few sessions of people who were logged on with subscriptions, and they too are producing hundreds of what appeared to be on that necessary IP authenticate and get user calls.

### ISSUE THREAD

**Issue by jwoosnam** [Psychoanalytic-Electronic-Publishing/OpenPubArchive-Content-Server#149]
@nrshapiro I have finished work for the day, but I just had a random look at a few sessions of people who were logged on with subscriptions, and they too are producing hundreds of what appeared to be on that necessary IP authenticate and get user calls.
The session below did a few successful document reads yesterday, but has been producing IP authenticate and get user calls ever since!!!!
<img width="2182" alt="Screenshot 2022-02-07 at 20 28 50" src="https://user-images.githubusercontent.com/70103034/152866925-37aff576-a8f1-4c36-ad6e-ea370ab86f2d.png">

_Originally posted by @jwoosnam in https://github.com/Psychoanalytic-Electronic-Publishing/PaDS/issues/24#issuecomment-1031892481_

**Comment by nrshapiro** [comment]
I saw this last night and thought this was in response to my issue I submitted yesterday--but it was open and I had not submitted!  Just put it in: see #150

If you want to hop on the weekly call I have with Rickard, perhaps we can diagnose together.

(You might want to come up with a shorter descriptive title for this one.)

**Comment by nrshapiro** [comment]
@jwoosnam this should be fixed on both production and stage.  Let me know if you agree and I'll close this issue.

**Comment by jwoosnam** [comment]
Yes this looks to be fixed.  The number of GetUser and AuthenticateIP (Session) calls are now only when you would expect them.
